ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/v1/present.yml

```diff
@@ -3,67 +3,67 @@
 - name: "Collecting facts from CVP {{ inventory_hostname }}."
   tags: [always]
   arista.cvp.cv_facts:
-  register: CVP_FACTS
+  register: cvp_facts
 
 - name: "Execute pending tasks on {{ inventory_hostname }}"
   tags: [apply]
   arista.cvp.cv_task:
-    tasks: "{{ CVP_FACTS.ansible_facts.tasks }}"
+    tasks: "{{ cvp_facts.ansible_facts.tasks }}"  # noqa: args[module]
   when: execute_tasks|bool
 
 - name: "Create configlets on CVP {{ inventory_hostname }}."
   tags: [provision]
   arista.cvp.cv_configlet:
-    cvp_facts: "{{ CVP_FACTS.ansible_facts }}"
-    configlets: "{{ CVP_VARS.CVP_CONFIGLETS }}"
+    cvp_facts: "{{ cvp_facts.ansible_facts }}"
+    configlets: "{{ cvp_vars.cvp_configlets }}"
     configlet_filter: ["{{ configlets_prefix }}"]
-  register: CVP_CONFIGLETS_STATUS
+  register: cvp_configlets_status
 
 - name: "Execute any configlet generated tasks to update configuration on {{ inventory_hostname }}"
   tags: [apply]
   arista.cvp.cv_task:
-    tasks: "{{ CVP_CONFIGLETS_STATUS.data.tasks }}"
+    tasks: "{{ cvp_configlets_status.data.tasks }}"  # noqa: args[module]
```

Comment:
what are we silencing here? I could not find the error in the failed CI

Reply:
The line number is the name line
```
WARNING: roles/eos_config_deploy_cvp/tasks/v1/present.yml:22: args[module]: Elements value for option 'tasks' is of type <class 'str'> and we were unable to convert to dict: unable to evaluate string as dictionary (warning)
```

Reply:
false positive - as mentioned here https://ansible-lint.readthedocs.io/rules/args/#special-cases

Reply:
hmmm ok maybe ansible-lint does not report eveything  and only the first error as I cannot see it in the failed CI but good! thanks for the answer!